### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by Nilsson-Jacob**

### CODE REVIEW

File: wordFinder.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package etsn20lab4;
+import java.io.File;  // Import the File class
+import java.io.FileNotFoundException;  // Import this class to handle errors
+import java.util.Scanner; // Import the Scanner class to read text files
+
+class wordFinder{
+    public static void main(String[] args){
+        try {
```
Comment (EBusk) [inline wordFinder.java:8]: Try to only keep the necessary lines inside the try - catch statements

File: wordFinder.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            if(args[0].length()<1||args[1].length()<1){
```
Comment (EBusk) [inline wordFinder.java:9]: It is a bit unclear where the values in the args vector come from. Perhaps a comment would make it easier to understand

File: wordFinder.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            Scanner myReader = new Scanner(textFile);
+            while (myReader.hasNextLine()) {
+              String data = myReader.nextLine();
+              if(checkIfWord(data,word)){
+                System.out.println(data);
+              }
+            }
+            myReader.close();
+          } catch (FileNotFoundException e) {
+            System.out.println("An error occurred.");
```
Comment (EBusk) [inline wordFinder.java:26]: The error message "An error occurred" is very general. Perhaps you want to write something more specific such as "The file could not be found"

File: ducks.txt [hunk 4]
```diff
@@ -0,0 +1,256 @@
+DUCKS
+
+What are ducks?
+
+Ducks are birds.
```
Comment (EBusk) [inline ducks.txt:5]: Good text

**Review by EBusk — COMMENTED** [review]
Overall, great code! Just a few small comments

**Review by EBusk — APPROVED** [review]

**Comment by Nilsson-Jacob** [comment]
Good points

**Pull request merged**